test/functional/feature_evm.py

```diff
@@ -135,7 +135,16 @@ def run_test(self):
         assert_equal(int(self.nodes[0].eth_getBalance(eth_address)[2:], 16), 10000000000000000000)
         assert_equal(int(self.nodes[0].eth_getBalance(to_address)[2:], 16), 0)
 
-        # Get miner DFI balance before transaction
+        # Send tokens to burn address
+        burn_address = "mfburnZSAM7Gs1hpDeNaMotJXSGA7edosG"
+        self.nodes[0].importprivkey("93ViFmLeJVgKSPxWGQHmSdT5RbeGDtGW4bsiwQM2qnQyucChMqQ")
```

Comment:
What is this key for?

Reply:
It is the burn address private key. I think this is needed to allow accounttoaccount as it requires the destination to be owned by the wallet. Really need to add a bool to skip that requirement.

Reply:
Resolving.